Source/WebCore/html/parser/HTMLTreeBuilder.cpp

```diff
@@ -1336,40 +1375,50 @@ void HTMLTreeBuilder::processStartTag(AtomHTMLToken&& token)
     case InsertionMode::Text:
         ASSERT_NOT_REACHED();
         break;
-    case InsertionMode::TemplateContents:
-        if (token.name() == templateTag) {
-            processTemplateStartTag(WTFMove(token));
-            return;
-        }
-
-        if (token.name() == linkTag
-            || token.name() == scriptTag
-            || token.name() == styleTag
-            || token.name() == metaTag) {
+    case InsertionMode::TemplateContents: {
+        InsertionMode insertionMode = InsertionMode::TemplateContents;
+        switch (token.tagName()) {
+        case TagName::base:
```

Comment:
Where did base, basefont, bgsound, etc... come from?

Reply:
Per discussion, this makes us match the HTML spec more closely (though it's not a behavior change). I'll split this out; filed https://bugs.webkit.org/show_bug.cgi?id=244525 for this.